backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1758,3 +1758,96 @@ databaseChangeLog:
             columns:
               - column:
                   name: test_result_delivery
+  - changeSet:
+      id: add-spring-session-jdbc-support
+      author: emma@skylight.digital
+      comment: Add a table to back spring sessions.
+      changes: 
+        - createTable:
+              tableName: spring_session
+              remarks: Data store for user sessions, as created by SpringSession.
+              columns:
+                - column:
+                    name: primary_id
+                    type: char(36)
+                    remarks: The immutable internal key to identify this session.
+                    constraints:
+                      nullable: false
+                      primaryKey: true
+                      primaryKeyName: pk__spring_session
+                - column:
+                    name: session_id
+                    type: char(36)
+                    remarks: A unique, externally visible string that identifies this session.
+                    constraints:
+                      nullable: false
+                      unique: true
+                      uniqueConstraintName: uk__spring_session
+                - column:
+                    name: creation_time
+                    type: DATETIME
+                    remarks: The creation timestamp for the session id.
+                    constraints: 
+                      nullable: false
+                - column:
+                    name: last_access_time
+                    type: DATETIME
+                    remarks: The last time the session id was used.
+                    constraints:
+                      nullable: false 
+                - column:
+                    name: max_inactive_interval
+                    type: int
+                    remarks: The maximum inactive interval time before a session times out and is invalidated.
+                    constraints: 
+                      nullable: facilities
```

Comment:
?

Reply:
Definitely an incorrect autocomplete 🤦  thanks for catching that!